src/transform.js

```diff
@@ -91,14 +91,14 @@ export function transformCommonjs ( code, id, isEntry, ignoreGlobal, ignoreRequi
 		if ( !node ) return;
 		if ( node.type !== 'CallExpression' ) return;
 		if ( node.callee.name !== 'require' || scope.contains( 'require' ) ) return;
-		if ( node.arguments.length !== 1 || node.arguments[0].type !== 'Literal' ) return; // TODO handle these weird cases?
+		if ( node.arguments.length !== 1 || (node.arguments[0].type !== 'Literal' && (node.arguments[0].type !== 'TemplateLiteral' || node.arguments[0].expressions.length > 0) ) ) return; // TODO handle these weird cases?
```

Comment:
Since this template literal case is now handled at least to some degree, the comment seems a bit misleading... it can probably go, eh?